AISKU/src/Initialization.ts

```diff
@@ -189,6 +189,14 @@ export class Initialization implements IApplicationInsights {
             _core = new AppInsightsCore();
             _self.core = _core;
 
+        
+            if (!config.connectionString) {
```

Comment:
We talked about also adding a config to disable this (for large customers who don't want to be annoyed by this message)

Reply:
I will create another PR to add the config! Thanks Nev!